esp/src/.vscode/tasks.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
@GordonSmith did you mean to include this file?

Reply:
Yes